scale/job/execution/running/tasks/base_task.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, task_id, job_exe):
         # These values will vary by different task subclasses
         self._uses_docker = True
         self._docker_image = None
+        self._docker_params = job_exe.get_docker_params()
```

Comment:
This should be set to []. These Docker params are meant to only be used by the job task, not the pre or post tasks. So the base class should have an empty list (no Docker params). You can do self._docker_params = job_exe.get_docker_params() in the JobTask constructor so it only applies to JobTasks. This is what is meant by "These values will vary by different task subclasses".